IBM/cloud-enterprise-examples
# [Docs] Improve website with automation and documentation

**Community Note**
Pull requests are always welcome, we are always thrilled to receive them and we do our best to process them quickly. If your pull request is not accepted on the first try, don't get discouraged!

- Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Ensure you have added or ran the appropriate tests for this pull request
- If the PR is unfinished, add `[WIP]` prefix to the pull request title and add the `do-not-merge` label.
- Include in the title the category of this pull request. The categories are: `[WIP]`, `[Feature]` or `[Fix]`. If the category is unknown use `[WIP]`, once the PR is ready to be merged replace `[WIP]` for the right category.
- Always assign label(s) to your PR.
- Any significant improvement should be documented as [a GitHub issue](https://github.com/IBM/cloud-enterprise-examples/issues) before anybody starts working on it.
- Make sure all your commits include a signature generated with `git commit -s`

**What this PR does / Why we need it?**

- Adds automation using Travis or GH Actions (or both) to generate the website in the `gh-pages` branch.
- (to be decided) Pack and publish the website in a docker container
- Add documentation to the repo explaining how to add content to the website
- Report the build/release status

**References**

- Fixes #000

**How I did it?**

- The previous website had a Travis to publish the content to a stating website, there is also a script executed manually to publish the content. Both will be reused to re-program the Travis or to create a GH Actions workflow
- The 

**How to verify it?**

- Running the scripts executed by Travis or GH Actions
- Publish a change to verify the CI/CD works

**Description for the changelog**

- Patterns/Solutions content from `att-cloudnative/ibmcloud-pattern-guide` is now in `website/` to centralize code and content allowing contributions from community
- Automation to generate the website when there is a change in the content
- (To be decided) Release the website as a docker container
